src/vcpkg/tools.cpp

```diff
@@ -174,10 +173,9 @@ namespace vcpkg
             return {std::move(output), expected_left_tag};
         }
 
-        auto error_prefix =
-            msg::format_error(msgUnexpectedToolOutput, msg::tool_name = tool_name, msg::path = exe_path).extract_data();
-        error_prefix.push_back('\n');
-        return {std::move(error_prefix) + std::move(output), expected_right_tag};
+        return std::move(msg::format_error(msgUnexpectedToolOutput, msg::tool_name = tool_name, msg::path = exe_path)
```

Comment:
Why does this all need to be wrapped in a `std::move()` -- isn't it already an rvalue?

Reply:
`LocalizedString::append` returns `LocalizedString&`, not `LocalizedString&&`.